single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java

```diff
@@ -856,7 +857,7 @@ private void addInfoComponentMetadata(GraphMetadata metadata, String componentTy
         }
     }
 
-    private void drawFeederInfo(String prefixId, String feederNodeId, List<Point> points, Element root,
+    private void drawFeederInfo(String prefixId, String feederNodeId, String equipmentId, String side, List<Point> points, Element root,
```

Comment:
Just a minor comment: to avoid having too many arguments here, could you replace the feeder attributes (nodeId, equipmentId, side) by the `FeederNode feederNode`?